MetaMask/metamask-extension
# Bump LavaMoat and SES

## Explanation

SES 0.18.7 is incompatible with Google Chrome 117, resulting in lockdown not fully working in the injected content script:

```
lockdown-run.js:17 Lockdown failed: TypeError: At intrinsics.Object.groupBy expected boolean not function
  at isAllowedPropertyValue (lockdown-install.js:1:53384)
  [...]
lockdown-more.js:99 Protecting intrinsics failed: ReferenceError: harden is not defined
  at lockdown-more.js:69:13
  [...]
```

This was fixed in SES 0.18.8. To make sure SES is up-to-date everywhere, we have to bump LavaMoat to the latest versions too.

## Manual Testing Steps

1. Run the extension locally.
2. Ensure there are no errors in the page console.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
